exp/api/v1beta1/azuremanagedmachinepool_webhook.go

```diff
@@ -42,6 +42,10 @@ func (r *AzureManagedMachinePool) Default(client client.Client) {
 	if r.Spec.Name == nil || *r.Spec.Name == "" {
 		r.Spec.Name = &r.Name
 	}
+
+	if r.Spec.ScaleSetPriority == nil {
```

Comment:
@CecileRobertMichon do we want to have capz to have a default opinion of this? The alternative would be to continue not including this configuration and defer to AKS for a default.

Reply:
+1 to defaulting because otherwise if AKS ever changed the default it wouldn't be backwards compatible if the field is immutable (I doubt they would do that for this particular setting but you never know)